tests/installation/accept_license.pm

```diff
@@ -29,10 +29,11 @@ use base 'y2_installbase';
 use y2_logs_helper qw(verify_license_has_to_be_accepted verify_license_translations accept_license);
 use testapi;
 use version_utils 'is_sle';
+use utils 'assert_screen_with_soft_timeout';
 
 sub run {
     my ($self) = @_;
-    assert_screen('license-agreement');
+    assert_screen_with_soft_timeout('license-agreement', timeout => 60, soft_timeout => 120, bugref => 'poo#63724');
```

Comment:
Hm, soft-fail reference to a progress ticket? Do you have a good idea for a fix? Maybe just bump the timeout regardless.

Reply:
Yeah, I have considered that as well, idea was to track those numbers in production. But as we accept this behavior, I think it's more consistent solution. Will just bump the timeout.